<launch>
   <arg name="input_image_topic"  default="usb_cam/image_raw"/>
   <arg name="img_motion_topic"   default="img_motion    "/>


   <!-- Motion detection node -->
   <node name="motiondetection" pkg="ohm_rrl_motiondetection2" type="ohm_motiondetection_node" output="screen">
      <param name="image_topic"          type="string"  value="$(arg input_image_topic)"/>   <!-- topic to subscribe camera              -->
      <param name="motion_image_topic"   type="string"  value="$(arg img_motion_topic)" />   <!-- topic for image publisher with markers --> 
      <param name="debug"                type="bool"    value="false"/>                <!-- remove output from node image          -->
     <!-- <param name="rect_topic"           type="string"  value="notification_rect"/> -->   <!-- publisher topic for rect in gui        -->
   </node>
   
</launch>